File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/resources/ResourcesModule.java
package com.androidcollider.easyfin.common.managers.resources;

import android.content.Context;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ResourcesModule {

    @Provides
    @Singleton
    ResourcesManager provideResourcesManager(Context context) {
        return new ResourcesManager(context);
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/resources/ResourcesModule.kt
package com.androidcollider.easyfin.common.managers.resources

import android.content.Context
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

/**
 * @author Ihor Bilous
 */
@Module
class ResourcesModule {
    @Provides
    @Singleton
    fun provideResourcesManager(context: Context): ResourcesManager {
        return ResourcesManager(context)
    }
}